web/src/js/components/Dashboard/__tests__/UserMenuComponent.test.js

```diff
@@ -571,36 +572,106 @@ describe('User menu component: sparkly search intro button', () => {
   })
 })
 
-describe('User menu component: campaign reopen button', () => {
-  it('displays the campaign reopen button when the showCampaignReopenButton prop is true', () => {
+describe('User menu component: video ads components', () => {
+  it('displays the video ad icon when enable video ads is set to true', () => {
     const mockProps = getMockProps()
-    mockProps.showCampaignReopenButton = true
+    process.env.REACT_APP_ENABLE_VIDEO_ADS = 'true'
```

Comment:
How are these tests working when the feature flag is incorrect? And, rather than set env vars, mock the feature flag function.

Reply:
sorry tests weren't passing, i forgot to add the file to my commit 🤦